src/components/ImageView/RegionView/RegionViewComponent.tsx

```diff
@@ -739,7 +739,7 @@ export class RegionViewComponent extends React.Component<RegionViewComponentProp
         }
 
         let cursor: string;
-        if (regionSet.mode === RegionMode.CREATING || AppStore.Instance?.activeLayer === ImageViewLayer.DistanceMeasuring) {
+        if ((regionSet.mode === RegionMode.CREATING || AppStore.Instance?.activeLayer === ImageViewLayer.DistanceMeasuring) && !frame.isPreview) {
```

Comment:
Currently we can draw regions with the menu. I think we either disable drawing regions or keep the crosshair cursor style for the `CREATING` region mode.